src/lecturer/components/studentListView/StudentListView.tsx

```diff
@@ -85,31 +93,42 @@ export function StudentListView(props: StudentListViewProps) {
         setOrderBy(property);
     };
 
+    const handleButtonClick = () => {
+        copy("http://localhost:3001/" + store.link, {
+            message: 'Press #{key} to copy',});
+    }
+
     return (
-        <TableContainer component={Paper} className={classes.root}>
-            <div><a target="_blank" rel="noreferrer" href={"http://localhost:3001/" + store.link}>LectureLink: http://localhost:3001/{store.link}</a></div>
-            <Table aria-label="tabela z listą studentów">
-                <StudentListHead
-                    order={order}
-                    orderBy={orderBy}
-                    onRequestSort={handleRequestSort}
-                    cells={headCells}
-                />
-                <TableBody>
-                    {stableSort(studentList, getComparator(order, orderBy))
-                        .map((row, index) => {
-                            return (
-                                <TableRow key={row.id} className={classes.row}>
-                                    <TableCell>{row.orderIndex}</TableCell>
-                                    <TableCell>{row.nick}</TableCell>
-                                    <TableCell>{row.name}</TableCell>
-                                    <TableCell>{row.surname}</TableCell>
-                                </TableRow>
-                            );
-                        })}
+        <>
+            <TableContainer component={Paper} className={classes.root}>
+                <Table aria-label="tabela z listą studentów">
+                    <StudentListHead
+                        order={order}
+                        orderBy={orderBy}
+                        onRequestSort={handleRequestSort}
+                        cells={headCells}
+                    />
+                    <TableBody>
+                        {stableSort(studentList, getComparator(order, orderBy))
+                            .map((row, index) => {
+                                return (
+                                    <TableRow key={row.id} className={classes.row}>
+                                        <TableCell>{row.orderIndex}</TableCell>
+                                        <TableCell>{row.nick}</TableCell>
+                                        <TableCell>{row.name}</TableCell>
+                                        <TableCell>{row.surname}</TableCell>
+                                    </TableRow>
+                                );
+                            })}
+                    </TableBody>
+                </Table>
+            </TableContainer>
+            <div className={classes.buttonDiv}>
+                <Button className={classes.btn} variant="contained" color="primary"  onClick={handleButtonClick}>
+                        COPY LINK
+                </Button>
+            </div>
```

Comment:
When we said about moving it to another component we meant splitting it into two different files. Just like Student list has heading in [StudentListHead.tsx](https://github.com/michal367/OI-Project/blob/master/src/lecturer/components/studentListView/StudentListHead.tsx) and body in StudentListView.tsx 
![image](https://user-images.githubusercontent.com/26445545/116275596-11bc3200-a784-11eb-8646-66365db9fede.png)